packages/framework/src/Foundation/ConsoleKernel.php

```diff
@@ -18,6 +18,16 @@ protected function bootstrappers(): array
 
         array_splice($bootstrappers, 5, 0, LoadYamlConfiguration::class);
 
-        return $bootstrappers;
+        // Since we store our application config in `app/config.php`, we need to replace
```

Comment:
Since the removed config files like view and cache are now inlined and dependent on a service provider registration we should put that in the bootstrapper.

Reply:
Resolved in https://github.com/hydephp/develop/pull/978